target/linux/lantiq/image/Makefile

```diff
@@ -527,9 +559,10 @@ endef
 TARGET_DEVICES += zte_h201l
 
 define Device/zyxel_p-2601hn
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := P-2601HN-Fx
```

Comment:
Sorry, just found during this iteration: "Fx" might belong to VARIANT? @dangowrt ?

Reply:
I think this refers to the F1 and F3 model.
See https://openwrt.org/toh/zyxel/zyxel_p2601hnfx

Reply:
Then we should use `DEVICE_VARIANT := F1/F3` here.